console/console-init/ui/src/modules/iot-device/components/ViewMetadata/MetadataReviewRow.tsx

```diff
@@ -7,7 +7,9 @@ import React, { useState } from "react";
 import { StyleSheet, css } from "aphrodite";
 import { AngleRightIcon, AngleDownIcon } from "@patternfly/react-icons";
 import { GridItem, TextInput, Grid, Button } from "@patternfly/react-core";
-import { MetaDataReviewRows } from "./MetaDataReviewRows";
+import { MetadataReviewRows } from "./MetadataReviewRows";
+import { uniqueId } from "utils";
```

Comment:
There is a unique id method available in lodash, I believe we should use that everywhere. @ajaypratap003 @rkpattnaik780 @drcoolsanjeev

Reply:
There are multiple unique id generators used across metadata components. uniqueId from Utils, uniqueId from lodash and uniqueId from PF. I'd suggest we use a common uniqueId generator everywhere. According to me, with so many uniqueId generators, we don't need the one in Utils, and the PF's unique id comes with a pf prefix which doesn't make sense in all use cases. So my suggestion would be to use the lodash one. @ajaypratap003 @drcoolsanjeev @rkpattnaik780 @suyash-naithani

Reply:
@anukritijha yes agree, we should use lodash uniqueId.